### PR TITLE
add to ctxt all active attrs for explicitAttrs when is a jexl

### PR DESCRIPTION
**Pull request by AlvaroVega**
Add to context for explictAttrs jexl evaluation the following missed concepts:
- active attributes with expressions (and also indentified by its object_id)
- Measures identified by its object_id)

Check if explicitAttr list contains attributes identified by its object_id

- [x] Add test about explicitAttrs with jexl and multientity: https://github.com/telefonicaid/iotagent-node-lib/issues/1351

### CODE REVIEW

File: test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js [hunk 1]
```diff
@@ -1311,7 +1425,7 @@ describe('Java expression language (JEXL) based transformations plugin', functio
 
 describe('Java expression language (JEXL) based transformations plugin - Timestamps', function () {
     beforeEach(function (done) {
-        logger.setLevel('FATAL');
+        logger.setLevel('DEBUG');
```
Comment (mapedraza) [inline test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js:1428]:
```suggestion
        logger.setLevel('FATAL');
```

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,2 +1,3 @@
+- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expresison (#1349)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expresison
```

(In general we are not self-linking PRs in the changelog for 3.1.0)

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -1,2 +1,3 @@
+- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expression
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: add missed active attributes (with expressions) for ctxt for evaluate explicitAttrs expression (#1351)
```

given this comment https://github.com/telefonicaid/iotagent-node-lib/issues/1351#issuecomment-1534203621

**Review by fgalan — APPROVED** [review]
LGTM

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**